graph/src/data/store/scalar.rs

```diff
@@ -115,23 +126,23 @@ impl Add for BigDecimal {
     type Output = Self;
 
     fn add(self, other: Self) -> Self {
-        Self::from(self.0.add(other.0))
+        Self::from(self.0.add(other.0)).normalized()
     }
 }
 
 impl Sub for BigDecimal {
     type Output = Self;
 
     fn sub(self, other: Self) -> Self {
-        Self::from(self.0.sub(other.0))
+        Self::from(self.0.sub(other.0)).normalized()
```

Comment:
`from` already normalizes - so all these math operations now normalize twice.

Reply:
True, I got confused here and thought we weren't normalizing the output. Removed this commit.